configs/eywa_development.yml

```diff
@@ -4,6 +4,7 @@ service:
   device_port: 8081
   pid_file: {{ .eywa_home }}/tmp/pids/eywa_development.pid
   assets: {{ .eywa_home }}/assets
+  header_path: {{ .eywa_home }}/headers
```

Comment:
I actually think `templates`is more appropriate now. Because, header files is subject to `c` or `cpp`, while js and py don't have header files.